apisix/cli/ops.lua

```diff
@@ -728,14 +730,16 @@ local function start(env, ...)
     local pid = util.read_file(pid_path)
     pid = tonumber(pid)
     if pid then
-        local lsof_cmd = "lsof -p " .. pid
-        local res, err = util.execute_cmd(lsof_cmd)
-        if not (res and res == "") then
-            if not res then
-                print(err)
-            else
-                print("APISIX is running...")
-            end
+        local signone = 0
+
+        local ok, err, err_no = signal.kill(pid, signone)
+        if ok then
+            print("APISIX is running...")
+
+            return
+        -- no such process
```

Comment:
The comment is in the wrong place?

Reply:
Here want to show meaning of  `errno.ESRCH` is `no such process`, so place upon it.

Need to remove it?